### PR TITLE
Added wallpaper support for sway

### PR DESCRIPTION
**Pull request by regenbogencode**
This pull-request adds support for setting wallpapers on sway.

### CODE REVIEW

File: pokemonterminal/wallpaper/adapters/sway.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from os import environ
+from subprocess import run
+from . import WallpaperProvider as _WProv
+
+
+class SwayProvider(_WProv):
+    def change_wallpaper(path: str):
+        run(["swaymsg", f"output * background {path} fill"], check=True)
```
Comment (sylveon) [inline pokemonterminal/wallpaper/adapters/sway.py]: Chances are this won't work with paths that have spaces

Comment (sylveon) [reply pokemonterminal/wallpaper/adapters/sway.py]: Also, Python will emit the command as this: `swaymsg "output * background aaa.jpg fill"`, but I have no idea if swaymsg will handle those quotes correctly.

Comment (sylveon) [inline pokemonterminal/wallpaper/adapters/sway.py]: Chances are this won't work with paths that have spaces
Also, Python will emit the command as this: `swaymsg "output * background aaa.jpg fill"`, but I have no idea if swaymsg will handle those quotes correctly.

Comment (regenbogencode) [reply pokemonterminal/wallpaper/adapters/sway.py]: Those quotes are actually needed for swaymsg; below is a gif of me using swaymsg and pokemon-terminal to update my sway wallpaper.



PokemonTerminal

**Comment by sylveon** [comment]
Thank you for the contribution!

**Comment by regenbogencode** [comment]
You are very welcome, I'm glad I could contribute to your awesome project!

**Pull request merged**